src/Reflection/AdditionalConstructorsExtension.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Reflection;
+
+interface AdditionalConstructorsExtension
```

Comment:
Would be nice to add a PHPDoc description similar to https://github.com/phpstan/phpstan-src/blob/979175ae6fa9ad2c3a2a5f228ac0449cded7900d/src/PhpDoc/StubFilesExtension.php#L5-L20 (also `@api` is needed)